packages/contracts-bedrock/contracts/L1/L1ERC721Bridge.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { ERC721Bridge } from "../universal/ERC721Bridge.sol";
+import { IERC721 } from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import { L2ERC721Bridge } from "../L2/L2ERC721Bridge.sol";
+import { Semver } from "../universal/Semver.sol";
+
+/**
+ * @title L1ERC721Bridge
+ * @notice The L1 ERC721 bridge is a contract which works together with the L2 ERC721 bridge to
+ *         make it possible to transfer ERC721 tokens from Ethereum to Optimism. This contract
+ *         acts as an escrow for ERC721 tokens deposited into L2.
+ */
+contract L1ERC721Bridge is ERC721Bridge, Semver {
+    /**
+     * @notice Mapping of L1 token to L2 token to ID to boolean, indicating if the given L1 token
+     *         by ID was deposited for a given L2 token.
+     */
+    mapping(address => mapping(address => mapping(uint256 => bool))) public deposits;
+
+    /**
+     * @custom:semver 1.0.0
```

Comment:
Should this be 1.0.0? Or 0.0.1?

Reply:
It is deployed with `1.0.0`, but willing to revert if you have a strong preference